model/countries/IN/IN-formatting-rules.yaml

```diff
@@ -0,0 +1,35 @@
+formatting-rules:
+  address:
+  - street-address-alternative-1
+  - separator: "\n"
+  - postal-code
```

Comment:
This does not correspond to libaddressinput nor my meeting notes.

https://www.upu.int/UPU/media/upu/PostalEntitiesFiles/addressingUnit/indEn.pdf
https://en.wikipedia.org/wiki/Address

There seem to be multiple address formats but none of them put the postal code before the localit1

Reply:
Sorry I didn't realize I had sent this for review. The PR is ready now. 

I've followed the format presented in the Aadhaar form [1]. It also matches most of the address formats in the links you shared above.

[1] https://5.imimg.com/data5/SELLER/Doc/2021/4/JZ/CF/JJ/126263748/aadhaar-card.pdf